src/main/java/mate/academy/dao/jdbc/OrderDaoJdbcImpl.java

```diff
@@ -0,0 +1,221 @@
+package mate.academy.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.OrderDao;
+import mate.academy.exceptions.DataProcessingException;
+import mate.academy.lb.Dao;
+import mate.academy.model.Order;
+import mate.academy.model.Product;
+import mate.academy.util.ConnectionUtil;
+
+@Dao
+public class OrderDaoJdbcImpl implements OrderDao {
+
+    @Override
+    public Order create(Order order) {
+        String insertOrder = "INSERT INTO orders (user_id) VALUES (?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement insertOrderQueryStatement
+                        = connection.prepareStatement(insertOrder,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            insertOrderQueryStatement.setLong(1, order.getUserId());
+            insertOrderQueryStatement.executeUpdate();
+            ResultSet resultSet = insertOrderQueryStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                Long orderId = resultSet.getLong(1);
+                order.setId(orderId);
+                insertOrdersProducts(order, connection);
```

Comment:
it's better not to open 2 statements at the same time, you can think of the way to close previous one before opening new in insertOrdersProducts